src/lib/bits/tooltip/components/TooltipTrigger.svelte

```diff
@@ -23,6 +23,7 @@
 		use:melt={builder}
 		{...$$restProps}
 		{...attrs}
+		type="button"
```

Comment:
```suggestion
```

I think we leave this up to the dev to add here if they wish. Imagine you wanted to add a tooltip to a button that submits a form, with the tooltip saying "Submit form" or something. We don't want to prevent the user from doing something like this.

Reply:
makes sense here. Done.